mmv1/products/cgc/terraform.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Snippet: !ruby/object:Overrides::Terraform::ResourceOverride
+    cgc_only: true
+    examples:
+    # SQL
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_database_instance_sqlserver"
+        primary_resource_type: "google_sql_database_instance"
+        primary_resource_id: "instance"
+        vars:
+          database_instance_name: "sqlserver-instance"
```

Comment:
@betsy-lichtenberg are you okay with the name `sqlserver-instance` showing up in the snippet? There was nowhere to define the variables in https://github.com/GoogleCloudPlatform/magic-modules/pull/5579 so I made up something, but let me know if you have something better in mind.

Reply:
That works for me.